containers/Dashboard/Dashboard.js

```diff
@@ -1,15 +1,147 @@
-import * as React from 'react';
-import Map from '../../components/Map/Map';
+import React, { useState, useEffect } from 'react';
 import HeaderNav from '../../components/HeaderNav/HeaderNav';
+import Map from '../../components/Map/Map';
+import FilterByDate from '../../components/FilterByDate/FilterByDate';
+import FilterByActivity from '../../components/FilterByActivity/FilterByActivity';
 import ShortEventCardContainer from '../../components/ShortEventCardContainer/ShortEventCardContainer';
-import { StyleSheet, View } from 'react-native';
+import { StyleSheet, View, Text } from 'react-native';
 
 export default function Dashboard({ navigation }) {
+  const [ events, setEvents ] = useState([])
+  const [ filterValue, setFilterValue ] = useState('')
+
+  let mockEvents = [
+    {
+      id: 1,
+      date: 20200407,
+      time: 1700,
+      unix_time: 1586278800,
+      activity: 'Volleyball',
+      location: 'Wash Park',
+      lat: 39.700,
+      long: -104.973,
+      duration: 'Could be a min and max input from the user -- ex. 15 min 120 max',
+      notes: 'If the weather is poor, meet here instead',
+      max_players: 10,
+      players_going: 5,
+      skill_level: 'Professional',
+      equipment: [
+        {
+          name: 'Net',
+          still_needed: false
+        },
+        {
```

Comment:
Just need to connect this to the API as soon as we can. After discussing it, we should prob hold the fetched event data in the state of the dashboard component.